### DISCUSSION TITLE
It doesn't work for iboss

### DISCUSSION THREAD

**Discussion by seeddddd** [3kh0/ext-remover#436] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I tried all of them and they don't work for iboss

**Comment by Aka-but-nice** [comment]
Try tinyurl.com/goofboss then follow the steps on New Point Blank

**Reply by seeddddd** [reply]
i tried it but all my tabs refresh and then it doesn't work

**Reply by Aka-but-nice** [reply]
Hmm

**Reply by Aka-but-nice** [reply]
Would you send a video, for documentation?

**Reply by seeddddd** [reply]
https://watch.screencastify.com/v/7UhbBugoIYqyr6drEO57

**Comment by BroJac5246** [comment, marked as answer]
iBoss will automatically restart itself. I created a modified version [here](https://gist.github.com/BroJac5246/4f8d7a9b5354f5b8edc41480ce02e301) that kills iBoss every one second and it seems to work fine. Make sure you keep both tabs (the 'block' page and about:blank) open, though.

**Reply by seeddddd** [reply]
i have tried this and when i visit the link, the iboss block screen shows up for half a second and then it disappears 
how do you use this

**Reply by BroJac5246** [reply]
It's because GitHub cut off the link. Try this:
`chrome-extension://kmffehbidlalibfeklaefnckpidbodff/restricted.html?re=1&bc=%3Cstyle%3Ep%20{color:%20rgba(0,0,0,0);}%20a%20{color:%20rgba(0,%200,%200,%201);}.left-content%20{display:%20none%20!important;}img%20{display:%20none%20!important;}form%20{display:%20none%20!important;}%20h2%20{display:%20none;}h3%20{display:%20none;}%3C/style%3E%3Ca+href=%22about:blank%22%20rel=%22opener%22%20target=%22_blank%22%3EClick%20here,%20then%20run%20the%20bookmarklet%20on%20the%20new%20tab.%3C/a%3E`

**Reply by BroJac5246** [reply]
@seeddddd did that work?

**Reply by goetzhold23** [reply]
its not working for me @BroJac5246

**Reply by BroJac5246** [reply]
@goetzhold23 Did you follow these steps:
[Demo.webm](https://user-images.githubusercontent.com/104146239/216661513-a6dfb1f4-d47d-41e5-9eef-adfae3972d83.webm)
The IBoss URL is `chrome-extension://kmffehbidlalibfeklaefnckpidbodff/restricted.html?re=1&bc=%3Cstyle%3Ediv%20{background:%20white%20!important;}%20p%20{color:%20rgba(0,0,0,0);}%20a%20{color:%20rgba(0,%200,%200,%201);}.left-content%20{display:%20none%20!important;}img%20{display:%20none%20!important;}form%20{display:%20none%20!important;}%20h2%20{display:%20none;}h3%20{display:%20none;}%3C/style%3E%3Ca+href=%22about:blank%22%20rel=%22opener%22%20target=%22_blank%22%3EClick%20here,%20then%20run%20the%20bookmarklet%20on%20the%20new%20tab.%3C/a%3E` and the iBreak (my version of the exploit) URL is [here](https://gist.github.com/BroJac5246/4f8d7a9b5354f5b8edc41480ce02e301)-- triple-click the code and drag it to the bookmarks bar.

**Reply by BroJac5246** [reply]
@seeddddd @goetzhold23 any luck?

**Reply by goetzhold23** [reply]
@BroJac5246 it worked once but now its not working anymore

**Reply by BroJac5246** [reply]
> @BroJac5246 it worked once but now its not working anymore

Is the link working, and if so does the menu appear when you run the bookmarklet on the about:blank tab?

**Reply by goetzhold23** [reply]
@BroJac5246 yeah the bookmarklet opens on it but when i click soft disable or hard disable it does nothing, but it worked fine once.

**Reply by goetzhold23** [reply]
now it is working. i think its like on and off for working. its probably something on my end

**Reply by BroJac5246** [reply]
> now it is working. i think its like on and off for working. its probably something on my end

Keep in mind that if the tab is kept open in the background, Chrome will eventually attempt to hibernate it, which could be part of the problem.

**Reply by goetzhold23** [reply]
ok, thanks
